analysers/analyser_merge_defibrillateurs_FR_basrhin.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Nicolas Bétheuil 2019                                      ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, GeoJSON, Load, Mapping, Select, Generate
+
+
+class Analyser_merge_defibrillateurs_FR_basrhin(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        self.missing_official = {"item":"8321", "class": 10, "level": 3, "tag": ["merge"], "desc": T_(u"Defibrillator not integrated") }
```

Comment:
Use a new class at each analyser, 10, 20, 30...

Reply:
what is the meaning of class ? I have put the same on all analysis, in each country there will be a different, maybe some data set will cover back some others, one class by analyser ?

Reply:
Since the analyse is not the same for each area, you can reuse the same class. But for practical it does not help to reuse the class.

Please, use 10, 20, 30, not 1, 2, 3, 4. We keep the last digit for other usage in merge analyser (missing_official, missing_osm, update...).

Reply:
I will get more that 10 analyzers, ok to go up to 100 ?

Reply:
yes